### DISCUSSION TITLE
Tabs under Members are empty

### DISCUSSION THREAD

**Discussion by ibaranov91** [dani-garcia/vaultwarden#4822] [Q&A]
Hi everyone,

Maybe someone has faced a similar issue and knows how to solve it. I've created an organization and synced users using the directory connector. The "All" tab works just fine, but the "Invited", "Needs confirmation", and "Revoked" tabs, while displaying the number on the tab, show up as empty.

![CleanShot 2024-08-07 at 11 35 07](https://github.com/user-attachments/assets/a98c0365-a5f4-48c3-9953-b91e750e9ed9)

**Comment by stefan0xC** [comment]
Can you provide more infos? I mean if the server logs don't have anything, the web console (open with <kbd class="kbd docutils literal notranslate">Ctrl</kbd> + <kbd class="kbd docutils literal notranslate">Shift</kbd> + <kbd class="kbd docutils literal notranslate">K</kbd> with Firefox or <kbd class="kbd docutils literal notranslate">Ctrl</kbd> + <kbd class="kbd docutils literal notranslate">Shift</kbd> +  <kbd class="kbd docutils literal notranslate">J</kbd> with Chrome) should show what went wrong (e.g. if an endpoint is missing or the data you got back is invalid, or that the web-vault threw an exception).

**Reply by ibaranov91** [reply]
Hi @stefan0xC, I checked it before, here are the errors I'm seeing in the Console, but I'm not sure they're related:
![CleanShot 2024-08-09 at 10 54 03@2x](https://github.com/user-attachments/assets/8ee6c1a3-b3ec-4fa2-87bd-26a01f023974)

However I found a workaround. If I open the "Invited", "Needs confirmation", and "Revoked" tab and then type something in the "Search Members" field and then delete what I typed right away users do show up after that. No matter on which tab I do this trick they all start working simultaneously after that.

**Reply by stefan0xC** [reply]
Are you using `v2024.5.1b` of the web vault (latest release)? Because I think such a bug was fixed in a newer version? (Not sure which.) So it might be fixed in the `testing` image already.

**Reply by ibaranov91** [reply]
We're on the latest version I believe
![CleanShot 2024-08-09 at 11 19 20@2x](https://github.com/user-attachments/assets/1130b750-a744-4d10-a723-493d841d42cc)

**Reply by BlackDex** [reply]
Please try the`testing` tagged images as @stefan0xC suggested and see if that solves the issue.
